### PR TITLE
Fix the cleanup

### PR DESCRIPTION
**Pull request by TechnichalDoge**
#4

### CODE REVIEW

File: lua/weapons/gmod_tool/stools/hoverball_spawner.lua [hunk 1]
```diff
@@ -12,6 +12,8 @@ TOOL.ClientConVar[ "model" ] = "models/dav0r/hoverball.mdl"
 
 local ConVarsDefault = TOOL:BuildConVarList()
 
+cleanup.Register( "Marums Hovers" ) -- https://github.com/Facepunch/garrysmod/blob/master/garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stools/hoverball.lua
```
Comment (plally) [inline lua/weapons/gmod_tool/stools/hoverball_spawner.lua]:
```suggestion
cleanup.Register( "Marums Hovers" )
```

**Review by plally — CHANGES_REQUESTED** [review]
looks good only 1 issue I see

File: lua/weapons/gmod_tool/stools/hoverball_spawner.lua (hunk 1 above)
Comment (TechnichalDoge) [reply lua/weapons/gmod_tool/stools/hoverball_spawner.lua]: Done

File: lua/weapons/gmod_tool/stools/hoverball_spawner.lua [hunk 2]
```diff
@@ -27,6 +29,7 @@ function TOOL:LeftClick( trace )
         if (IsValid(trace.Entity)) then
             local weld = constraint.Weld( entity, trace.Entity, 0, trace.PhysicsBone, 0, true , false )
         end
+            self:GetOwner():AddCleanup( "Marums Hovers", entity )
```
Comment (plally) [inline lua/weapons/gmod_tool/stools/hoverball_spawner.lua]: i think this is this over indented

Comment (TechnichalDoge) [reply lua/weapons/gmod_tool/stools/hoverball_spawner.lua]: fixed

**Review by sarahsturgeon — APPROVED** [review]

**Review by sarahsturgeon — APPROVED** [review]
LGTM - approving the code only, someone needs to verify that they've tested it.

**Comment by TechnichalDoge** [comment]
I tested it

**Comment by TechnichalDoge** [comment]
Retesting it now, because ill get images

**Review by plally — APPROVED** [review]

**Comment by TechnichalDoge** [comment]
[https://www.youtube.com/watch?v=OpAhv4RDOrk](url)

**Pull request merged**